electron-builder.json

```diff
@@ -93,6 +101,19 @@
       "LSFileQuarantineEnabled": true
   }
   },
+  "mas": {
+    "hardenedRuntime": false,
```

Comment:
The app crashes on opening when this is enabled. I think it has to do with accessing other files or something?
It might just not work for Electron at all right now.

Reply:
that's weird, as it works with self-publishing. I'd suggest to at least add a ticket to investigate further at some point.

Reply:
I agree with Guillermo. We *should* be able to enable this and investigating why that's not the case is important, but I'd be fine with shipping with it disabled for now.